tests/unit_tests/test_prodigal.py

```diff
@@ -0,0 +1,120 @@
+    "NODE_1076_length_13288_cov_223.745",
+    "NODE_1080_length_13246_cov_222.788",
+    "NODE_1086_length_13138_cov_224.524",
+    "NODE_22_length_427389_cov_223.136",
+    "NODE_24_length_372190_cov_224.35",
+    "NODE_26_length_356957_cov_224.046",
+    "NODE_28_length_331107_cov_224.832",
+    "NODE_30_length_320616_cov_227.122",
+    "NODE_31_length_318595_cov_223.603",
+]
+
+# TODO mock ORFs file called by prodigal on test data
+# fpath = os.path.join(os.path.dirname(__file__)"/tests/data/test_prots.out"
+
+# fpath = os.path.join(os.path.dirname(os.path.dirname(__file__)), "data/test_prots.out")
+
+# # @patch("test_prodigal.prodigal.get_versions", return_value="2.6")
+# def test_contigs_from_headers():
+#     with patch("test_prodigal.prodigal.get_versions", return_value="2.0"):
+#         out = prodigal.contigs_from_headers(fpath=fpath)
+#         assert len(out) == 75390
+#         assert type(out) is dict
+
+#     with patch("test_prodigal.prodigal.get_versions", return_value="2.6"):
+#         out = prodigal.contigs_from_headers(fpath=fpath)
+#         assert len(out) == 75390
+#         assert type(out) is dict
+
+
+# # @patch("test_prodigal.prodigal.get_versions", return_value="2.0")
+# def test_orf_records_from_contigs():
+#     with patch("test_prodigal.prodigal.get_versions", return_value="2.0"):
+#         out = prodigal.orf_records_from_contigs(contigs=contigs, fpath=fpath,)
+#         # assert len(out) == 4414
+
+#     with patch("test_prodigal.prodigal.get_versions", return_value="2.6"):
+#         out = prodigal.orf_records_from_contigs(contigs=contigs, fpath=fpath,)
+#         assert len(out) == 4414
+
+
+@patch("builtins.open", new_callable=mock_open, read_data="Contents of a fasta file")
+# See: https://stackoverflow.com/a/34677735/12671809 it can also be expanded
+# as https://queirozf.com/entries/python-unittest-examples-mocking-and-patching#patch-open-file
+# @patch("os.path.exists", return_value=True)
+@patch("os.path.getsize", return_value=2 * 1024 * 1024)
+@patch("autometa.common.external.prodigal.annotate_parallel")
+@patch("autometa.common.external.prodigal.annotate_sequential")
+def test_output(
+    patched_annotate_sequential,
+    patched_annotate_parallel,
+    patched_file_size,
+    patched_open,
+):
+    nucls_out = MagicMock(
+        return_value="path_to_nucs_out", name="nucleic acid_output_file"
+    )
+    prots_out = MagicMock(
+        return_value="path_to_prots_out", name="amino_acid_output_file"
+    )
+    prodigal.run(
+        assembly, nucls_out=nucls_out, prots_out=prots_out, force=True, parallel=False,
+    )
+    prodigal.run(
+        assembly, nucls_out=nucls_out, prots_out=prots_out, force=True, parallel=True,
+    )
+
+    assert patched_annotate_parallel.called is True
+    assert patched_annotate_sequential.called is True
+    assert patched_file_size.called is True
+    # assert patched_file_exists.called is True
+    assert patched_open.called is True
```

Comment:
The comparisons using `is` and `==` are not quite the same. See [here](https://www.tutorialspoint.com/difference-between-and-is-operator-in-python#:~:text=is%20and%20equals(%3D%3D),two%20variables%20are%20the%20same. "tutorials point page on difference between the is and == operator") for more details.

```suggestion
    assert patched_annotate_parallel.called == True
    assert patched_annotate_sequential.called == True
    assert patched_file_size.called == True
    # assert patched_file_exists.called == True
    assert patched_open.called == True
```